crd-convert/README.md

```diff
@@ -0,0 +1,79 @@
+# CRD conversion
+
+With Strimzi's CRD evolving, there needs to be a way to easily migrate existing CR configuration to the new CRD schema.
+For this purpose a custom Converter for each CRD needs to be implemented.
+
+There are two methods to implement:
+
+```
+public void convertTo(T instance, ApiVersion toVersion)
+public void convertTo(JsonNode node, ApiVersion toVersion)
+```
+
+The first one does conversion directly on the CR instance, which might later lead to reordered configuration output (think "git merge" problems).
+That's why we added the second method, doing the conversion directly on the CR' JsonNode instance, preserving the configuration output as much as possible. 
+
+The code already provides useful conversion methods, such as `move`, `delete`, `replace`. Hence you mostly just need to implement part of `replace` method; e.g. the actual replacement value.
```

Comment:
The idea is that you only need to implement replace when a node is being changed in a non-trivial way (as was the case for `Kafka.spec.kafka.listeners`).

Reply:
>> Hence you mostly just need to implement part of `replace` method; e.g. the actual replacement value when a CR is being changed in a non-trivial way (as was the case for Kafka.spec.kafka.listeners).

Like this?